common/uint160.go

```diff
@@ -94,6 +94,13 @@ func (f Uint160) MarshalJSON() ([]byte, error) {
 	return []byte("\"" + str + "\""), err
 }
 
+func (f Uint160) UnmarshalJSON(in []byte) (err error) {
+	if len(in) > 1 {
```

Comment:
We should return error if `len < 2`, otherwise user will see `err == nil` and thought unmarshal is successful...

Reply:
> We should return error if `len < 2`, otherwise user will see `err == nil` and thought unmarshal is successful...

I thought about that too but my way of thinking was that `UnmarshalJSON()` returns an error on an operation that fails (in your implementation of it) rather than signalling that there was no operation at all. meaning, if there is nothing to unmarshal, it shouldnt be an error.

your suggestion indicates that we should fail if there is nothing to unmarshal. eg. what if user intentionally `omitempty`s on his side?

Reply:
from `encoding/json`:
```
// Unmarshaler is the interface implemented by types
// that can unmarshal a JSON description of themselves.
// The input can be assumed to be a valid encoding of
// a JSON value. UnmarshalJSON must copy the JSON data
// if it wishes to retain the data after returning.
//
// By convention, to approximate the behavior of Unmarshal itself,
// Unmarshalers implement UnmarshalJSON([]byte("null")) as a no-op.
```

futhermore, from https://pkg.go.dev/encoding/json#Unmarshal:
```
To unmarshal JSON into a value implementing the Unmarshaler interface,
Unmarshal calls that value's UnmarshalJSON method, including when the input is a JSON null.
```

Reply:
my omitempty assumption was wrong. `omitempty` leaves out the field completely if empty. so it wouldnt be in the JSON anyway and thus wont attempt to unmarshal.

Reply:
done.